h2o-r/h2o-package/R/kvstore.R

```diff
@@ -145,6 +145,8 @@ h2o.getModel <- function(model_id) {
   MetricsClass <- paste0("H2O", model_category, "Metrics")
   # setup the metrics objects inside of model...
   model$training_metrics   <- new(MetricsClass, algorithm=json$algo, on_train=TRUE, on_valid=FALSE, on_xval=FALSE, metrics=model$training_metrics)
+  model$columns$domains <- json$output$domains
```

Comment:
Great!!!

@dipplestix - fiy - I believe you are working on something similar in python